js/validations/loginformValidation.js

```diff
@@ -0,0 +1,66 @@
+// validator.initialize();;
+const loginForm = document.querySelector("form");
+const handleLogin = (e) => {
+  e.preventDefault();
+  const formData = new FormData(loginForm).entries();
+  const { email, password } = Object.fromEntries(formData);
+  // console.log("keeeee");
+  const emailErrorMessage = validateEmail(email);
+  const passowrdErrorMessage = validatePassword(password, 6);
+
+  if (emailErrorMessage) {
+    // select the email form field message element
+    const emailErrorMessageElement = document.querySelector(
+      ".email.error-message"
+    );
+    // show email error message to user
+    emailErrorMessageElement.innerText = emailErrorMessage;
+  }
+
+  if (passowrdErrorMessage) {
+    // select the email form field message element
+    const passwordErrorMessageElement = document.querySelector(
+      ".password.error-message"
+    );
+    // show password error message to user
+    passwordErrorMessageElement.innerText = passowrdErrorMessage;
+  }
+  if ((emailErrorMessage === " ") & (passowrdErrorMessage === " ")) {
+    const saveEmail = JSON.stringify(email);
+    const savePassword = JSON.stringify(password);
+    console.log(saveEmail, savePassword);
+    localStorage.setItem("email", saveEmail);
+    localStorage.setItem("password", savePassword);
+    window.location.replace("http://127.0.0.1:5500/admin/admin.html");
```

Comment:
- Change redirect URL
- Add email & password in local storage as an object

Reply:
okay, How do you want me to change the redirect link can I use the links from netlify?

Reply:
Yes

Reply:
I have collected the requested changes please review them whenever you have time.
thanks